<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>interfaz</class>
 <widget class="QWidget" name="interfaz">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>537</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sistema Consulta Ventas</string>
  </property>
  <widget class="QLabel" name="lblBuscarFactura">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Mongolian Baiti</family>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Buscar N.º Factura</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="TxTBuscarFactura">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>10</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBuscar">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>20</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Buscar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCrear">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>60</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Nueva Venta</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnActualizar">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>110</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Editar Venta</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnEliminar">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>160</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Eliminar Venta</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="DetalleProducto">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>461</width>
     <height>251</height>
    </rect>
   </property>
   <property name="title">
    <string>Detalle Producto</string>
   </property>
   <widget class="QTableWidget" name="DetalleFactura">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>371</width>
      <height>192</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Código</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Nombre</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Precio</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Unidades</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Subtotal</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Total</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QGroupBox" name="DatosCliente">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>461</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Datos Cliente</string>
   </property>
   <widget class="QLineEdit" name="TxTNumFactura">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="lblNumFactura">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mongolian Baiti</family>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>N.º Factura</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblCodCliente">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mongolian Baiti</family>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>ID Cliente</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="TxTCodCliente">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="lblNombre">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mongolian Baiti</family>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Nombres</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="TxTNombre">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>30</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>60</number>
    </property>
   </widget>
   <widget class="QLabel" name="lblApellido">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>70</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mongolian Baiti</family>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Apellidos</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="TxTApellido">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>80</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>60</number>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnGuardar">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>250</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Guardar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLimpiar">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>290</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Limpiar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
